### DISCUSSION TITLE
Zed AI Claude 3.5 vs Anthropic Claude 3.5

### DISCUSSION THREAD

**Discussion by simicvm** [zed-industries/zed#16574] [Feature Requests]
Is there a difference in functionality / capability between Zed AI Claude 3.5 vs Anthropic Claude 3.5 (with the API key)?

**Comment by yasainet** [comment]
I was wondering about this too. I have read all the documentation, but could not quite understand the difference.

**Comment by SomeoneToIgnore** [comment]
Most of the models with `Zed` prefix are regular models proxied through Zed servers (so, different rate limits and other restrictions may apply).
This is made to allow all Zed Pro subscribers to use any of the models as they were the regular ones, without fiddling with extra keys.
So, there should not be any functional difference between Zed's and Anthropic's Claude 3.5 Sonnet.

Note that Zed also provides a totally new model thanks to partnership with Anthropic — this is a totally different beast compared to the "regular" Sonnet.
IIRC this is for now a unique offer, with no analogous public offer on Anthropic's side yet.

**Reply by simicvm** [reply]
Is the new model exclusive to Zed Pro subscribers? I don't see it in the model selector and I am not a Pro subscriber (at least not yet 🙂)

**Reply by SomeoneToIgnore** [reply]
I believe it's not a part of the free plan, but now thinking of it, I cannot be 100% certain it's available to all Pro subscribers either at the moment — sorry for the uncertainty, it's quite many moving pieces around now.

I believe, the new model will make it into both entire Pro and free tiers later, but it does not depend on me to guarantee.

**Reply by simicvm** [reply]
cool, thanks for the answers!

**Reply by crobbo** [reply]
Where's the information about Zed Pro? I can't see anything about it on your site.

**Reply by SomeoneToIgnore** [reply]
I think it's in closed preview right now, and will be properly announced later.

**Reply by crobbo** [reply]
![image](https://github.com/user-attachments/assets/48b8fd64-ad3a-4b54-8141-4c30fd6b2856)

Thanks. I seem to have access to this model?

**Reply by SomeoneToIgnore** [reply]
That's the same 3.5 Sonnet that either is routed through Zed services (above, does not need a dedicated Anthropic key) or used directly (below, needs an Anthropic key)

**Reply by crobbo** [reply]
OK cool, I'm not paying for anything so surprising I get access but I'm not complaining 😄
